spec.html

```diff
@@ -37808,7 +37808,10 @@ <h1>Proxy.revocable ( _target_, _handler_ )</h1>
         <p>The `Proxy.revocable` function is used to create a revocable Proxy object. When `Proxy.revocable` is called with arguments _target_ and _handler_, the following steps are taken:</p>
         <emu-alg>
           1. Let _p_ be ? ProxyCreate(_target_, _handler_).
-          1. Let _revoker_ be a new built-in function object as defined in <emu-xref href="#sec-proxy-revocation-functions"></emu-xref>.
+          1. Let _realm_ be the current Realm Record.
+          1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-proxy-revocation-functions"></emu-xref>.
+          1. Let _prototype_ be the intrinsic object %FunctionPrototype%.
+          1. Let _revoker_ be CreateBuiltinFunction(_realm_, _steps_, _prototype_, &laquo; [[RevocableProxy]] &raquo;).
```

Comment:
I'd inline `%FunctionPrototype%` in the call to `CreateBuiltinFunction`, cf. MakeArgGetter and MakeArgSetter.